router/io-spec.gobra

```diff
@@ -543,3 +543,48 @@ pure func uInfo(hopfields seq[io.IO_HF], currHFIdx int, consDir bool) set[io.IO_
 						hvfSet(hopfields[currHFIdx]) :
 						hvfSet(hopfields[currHFIdx-1])))
 }
+
+ghost
+decreases
+pure func ifsToIO_ifs(ifs uint16) option[io.IO_ifs]{
+    return ifs == 0 ? none[io.IO_ifs] : some(io.IO_ifs(ifs))
+}
+
+ghost 
+requires acc(sl.AbsSlice_Bytes(raw, 0, len(raw)), _)
+decreases
+pure func validPktMetaHdr(raw []byte) bool {
```

Comment:
In the future, when https://github.com/viperproject/gobra/pull/715, we should make this function `opaque` to avoid performance problems

Reply:
Thanks for the Info. I will keep an eye on that PR.